src/util/freehand/keysHeld.js

```diff
@@ -23,7 +22,7 @@ const keyCodes = {
 */
 export function keyDownCallback (e) {
   const eventData = e.detail;
```

Comment:
I get why this exists here, but I think we need to shift it up to be a bit more global. I'd like if we tracked which keys were down in `cornerstone-tool`s store, so other tools can use that information in their logic/callbacks without needing to do this themselves.

Reply:
Maybe each tool has an array of keycodes it cares about, and then the store/dispatcher tracks all keycodes that all added tools collectively care about? (that might be overkill, just noodling)

Reply:
Are these only fired if keyboard input is enabled? I'm having trouble getting `ctrl + click` behavior to work (I may not understand fully)

Reply:
Apologies for not making a note of this, but this functionality hasn't actually been ported yet, and isn't in use by the v3 tool. However, to explain: it does require the keyboard input to be enabled to function. This interface is just to highlight all nodes when ctrl is held and config.alwaysShowHandles is false. The ctrl + click to add and remove points uses the eventdata from the mouseDown event to check if ctrl was used. This later functionality is in the PR.

Please let me know if you wish to have any points clarified :).